node/coinstacks/solana/sample.env

```diff
@@ -7,3 +7,5 @@ INDEXER_URL=https://api.helius.xyz
 LOG_LEVEL=debug
 NETWORK=mainnet
 RPC_URL=https://mainnet.helius-rpc.com
+WS_URL=wss://atlas-mainnet.helius-rpc.com
```

Comment:
```suggestion
WS_URL=wss://mainnet.helius-rpc.com
```

Reply:
Fixed in [33a4238](https://github.com/shapeshift/unchained/pull/1053/commits/33a4238603fb52d1d0c78ab7639bde23265bee5f)